src/Apps/Feature/FeatureApp.tsx

```diff
@@ -1,20 +1,80 @@
 import React from "react"
 import { AppContainer } from "Apps/Components/AppContainer"
-import { Box } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
-
+import {
+  FeatureHeaderContainer as FeatureHeader,
+  FeatureFeaturedLink,
+  FeatureSet,
+} from "./Components"
 import { FeatureApp_feature } from "__generated__/FeatureApp_feature.graphql"
+import { Box, Sans, Spacer } from "@artsy/palette"
+import { FEATURED_LINK } from "./Components/__tests__/fixtures"
 
 interface FeatureAppProps {
-  children: React.ReactNode
   feature: FeatureApp_feature
 }
 
-const FeatureApp: React.FC<FeatureAppProps> = ({ children, feature }) => {
+const FeatureApp: React.FC<FeatureAppProps> = ({ feature }) => {
   return (
     <AppContainer maxWidth="100%">
-      <Box my={3}>{feature.name}</Box>
-      {children}
+      <Box height="100vh">
```

Comment:
I realize I need to offset by the header height here. I don't like this at all — it seems like it'd be prudent to eventually fix this in the layout so we don't have to hardcode all these calcs everywhere which is ugly.

Reply:
Def down to help clean this up, I'm not even sure how its possible without a `calc`

Reply:
You could avoid the `calc` by using `position: sticky` on the nav so that it placeholders correctly against the document. Or just keeping it fixed and using marginTop: NAV_HEIGHT a single time in the layout container (rather than having to specify it for every single page/app).